packages/aws-cdk-docs/src/getting-started.rst

```diff
@@ -865,6 +867,56 @@ encryption for the bucket:
     [2/2] UPDATE_COMPLETE     [AWS::CloudFormation::Stack] hello-cdk
     ✅  Deployment of stack hello-cdk completed successfully
 
+.. _using_cdk_init:
+
+Using cdk init to Create a Project

```

Comment:
I think this should be in the __tools__ topic in the description of the the various toolkit commands